src/scripts.js

```diff
@@ -28,13 +30,24 @@ const ingredientSidebar = document.querySelector("#ingredientSection")
 const userName = document.querySelector('#user-info');
 const favoritesView = document.querySelector('#favorites-view');
 const savedButton = document.querySelector('#saved-recipe-button');
+const saveRecipeButton = document.querySelector('#favorite-recipe-button')
+const homeButton = document.querySelector('#home-button')
+const submitButton = document.querySelector('#submit-search-button')
+const searchBar = document.querySelector('#search-bar')
 
 // ~~~~~~~~~~~~~~ Event Listeners ~~~~~~~~~~~~~~~~~~~~
 allRecipes.addEventListener('click', viewRecipeDetail);
 window.addEventListener('load', displayAllRecipes);
 window.addEventListener('load', displayWelcomeMessage);
-allRecipes.addEventListener('click', addRecipeToFavorites);
 savedButton.addEventListener('click', displayFavorites);
+saveRecipeButton.addEventListener('click', addRecipeToFavorites)
+homeButton.addEventListener('click', displayHomePage)
+favoritesView.addEventListener('dblclick', removeFromFavorites)
+submitButton.addEventListener('click', () => {
+    if(homeView) {searchForRecipe()}
```

Comment:
Ah, now the `let homeView = true` above makes sense! Also, love the double click to remove from favorites.